apps/zotonic_core/src/i18n/z_trans.erl

```diff
@@ -203,6 +211,10 @@ lookup_fallback(_Text, _Lang, _Context) ->
     undefined.
 
 
+is_string([H|T]) when is_integer(H), H >= 1 -> is_string(T);
+is_string([H|T]) when is_binary(H) -> is_string(T);
+is_string(_) -> false.
```

Comment:
This function always return `false` but expected a `true` or `false`.

Reply:
Good one :-)